install_packages.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+# Script for installing packages and removing apt cache
```

Comment:
Optional: I'm not convinced this is worth the complexity (debugging hassle when it goes wrong) vs just the conventional `RUN apt-get install -y --no-install-recommends ... && rm -r /var/{lib/apt/lists,cache/apt/archives}`. Indeed dockerfile linters actually check for this sort of thing 🤷 

I'm guessing it was included in minideb and we're keeping it for now so as not to require changing all the app Dockerfiles up front? (in which case that sounds fine!)

Reply:
Yes, this script is also included in minideb. It's a fairly simple script that (in my opinion) improves the readability of Dockerfiles. I believe we already have some application Dockerfiles that are using `install_packages`, so removing it would also mean some more work.

Reply:
It just occurred to me we need to fulfil any licence obligations if we're copying this from minideb. Minideb's licence is Apache 2.0.

Looks like the script is from here: https://github.com/bitnami/minideb/blob/6883052/mkimage#L209

If we're keeping it, the minimum we must do is including the original licence and copyright notice. There's no NOTICE file in minideb so we don't have to include that. And if we've made any significant modifications to the script then we have to summarise them.

Alternatively, we could drop the script and amend any Dockerfiles where we were using it.